test/unit/pipeline/M1PortAllocatorTest.cc

```diff
@@ -24,7 +24,7 @@ class M1PortAllocatorTest : public testing::Test {
       {0}, {1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}, {9}, {10}, {11}, {12}, {13}};
   // Representation of the M1 Firestorm Reservation Station Arrangement
   // std::pair<uint8_t, uint64_t> = <rsIndex, rsSize>
-  std::vector<std::pair<uint8_t, uint64_t>> rsArrangement = {
+  std::vector<std::pair<uint16_t, uint64_t>> rsArrangement = {
```

Comment:
What is the reasoning for the type widening? Are we going to have more than 256 indices?

Reply:
Type was updated to match the base class definition, where I think we are generally moving away from uint8_t. Maybe @jj16791 can provide more insight into the move awat from uint8_t generally?

Reply:
Needed for large theoretical processors with more than 255 RSs